### PR TITLE
Changes for real time storms functions

### PR DESCRIPTION
**Pull request by mhines-usgs**
The CCH team needs to support Noreasters/Winter storms so we have modified the service to allow for more than titles pertaining to Tropical Storms or Hurricanes. 

In Sept 2015, the NOAA server which hosts the NHC tracks changed, so this PR also includes changes related to that.

### CODE REVIEW

**Comment by mhines-usgs** [comment]
@jiwalker-usgs yo, can you take a peek before I merge?

File: R/realtime.service.R [hunk 1]
```diff
@@ -118,10 +118,11 @@ realtime.service = function(serviceEndpoint,attribute=NULL){
 
 extractInfoFromTitle = function(title) {
   # Hurricane Sandy Assessment of Potential Coastal-Change Impacts: NHC Advisory 29, 1100 AM EDT MON OCT 29 2012
-  match <- regexec("((Hurricane|[A-Za-z-]*[Tt]ropical \\w+) \\w+).*(NHC Advisory (\\d+), (.*))", title)
+  #match <- regexec("((Hurricane|[A-Za-z-]*[Tt]ropical \\w+|Nor'easter/Winter Storm) \\w+).*(NHC Advisory (\\d+), (.*))?", title)
```
Comment (mhines-usgs) [inline R/realtime.service.R]: I guess I could get rid of this, sorry. I can do that.

**Comment by mhines-usgs** [comment]
not sure why the travis CI failed, checking...

**Comment by jiwalker-usgs** [comment]
You probably want to up the version number in the DESCRIPTION as well.

**Comment by mhines-usgs** [comment]
@jiwalker-usgs good call, I'll do that (version number).

The Travis failure is related to the stupid BBOX thing we keep seeing. A test is failing apparently?

 checking examples ... ERROR
Running examples in ‘hazardItems-Ex.R’ failed
The error most likely occurred in:
> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: thumb.service
> ### Title: create thumbnail for an item
> ### Aliases: thumb.service
> 
> ### ** Examples
> 
> serviceEndpoint <- 'http://marine.usgs.gov/coastalchangehazardsportal/data/item/CAkR645'
> thumb.service(serviceEndpoint)
Error in bboxes[j, ] <- unlist(item.json$bbox) : 
  number of items to replace is not a multiple of replacement length
Calls: thumb.service -> getUniqueBBoxIDs
Execution halted

**Comment by jread-usgs** [comment]
tests work locally?

**Comment by mhines-usgs** [comment]
no @jread-usgs same issue.

**Comment by mhines-usgs** [comment]
Weird because we haven't changed anything related to the thumbservice...

**Comment by jread-usgs** [comment]
potentially some packages were updated between now and then.

**Comment by mhines-usgs** [comment]
on the server? aren't we fixing versions like AQCU?

**Comment by jread-usgs** [comment]
the server has manual (?) package installs, so you may be testing a different set of packages than exists on our rserve. travis and local are probably closest to the newest releases (travis will be completely current unless you get fancy w/ installs)

**Comment by mhines-usgs** [comment]
ah, right.. i think I can compare versions. Good point. So what fails on Travis doesn't necessarily mean we fail on the server... the last build passed and nothing changed since then, but you're right, likely the packages have...

**Pull request merged**